e2e/flux_test.go

```diff
@@ -0,0 +1,30 @@
+package e2e
+
+import (
+	"testing"
+
+	"github.com/kluctl/kluctl/v2/e2e/test_resources"
+)
+
+func TestFluxCommands(t *testing.T) {
+	t.Parallel()
+
+	k := defaultKindCluster1
+
+	p := &testProject{}
+	p.init(t, k, "simple")
+
+	defer p.cleanup()
+
+	test_resources.ApplyYaml("flux-source-crd.yaml", k)
+	test_resources.ApplyYaml("kluctl-crds.yaml", k)
+	test_resources.ApplyYaml("kluctl-deployment.yaml", k)
+
+	assertResourceExists(t, k, "default", "kluctldeployment/microservices-demo-test")
+
+	p.KluctlMust("flux", "suspend", "--namespace", "default", "--kluctl-deployment", "microservices-demo-test")
```

Comment:
can you also add a check after all the KluctlMust calls that verifies that the expected change has actually been performed? A simple KubectlYamlMust call + a few assertions (e.g. check if the spec.suspend field is true) should be enough.